docs/user/software/third-party/index.md

```diff
@@ -73,7 +73,7 @@ sudo eopkg bi --ignore-safety https://raw.githubusercontent.com/getsolus/3rd-par
 sudo eopkg it bitwig-studio*.eopkg;sudo rm bitwig-studio*.eopkg
 ```
 
-### OcenAudio
+### OceanAudio
```

Comment:
It's actually ocenaudio though (https://www.ocenaudio.com/)

Reply:
ffs